Gemfile

```diff
@@ -4,6 +4,7 @@ source "https://rubygems.org"
 
 gemspec
 
+gem "mini_mime", "~> 1.1"
```

Comment:
Can You please move it to the `allure-report-publisher.gemspec` since it is a runtime dependency and this tool is also published as a rubygem.

Reply:
ok I can move that. How about the changes in the `Gemfile.lock` ? Should I leave those as is or move it to the `PATH/specs` subsection? Sorry for the n00b question I am not a ruby developer.

Reply:
In context of this project, `Gemfile` and `Gemfile.lock` is mostly used for testing. `Gemfile.lock` shouldn't be touched manually really. You can move the definition to `.gemspec` and simply rerun `bundle install` or I can do it myself as well if You don't have the tools installed locally.